traceSdk.gradle

```diff
@@ -3,5 +3,5 @@ repositories {
 }
 
 dependencies {
-    implementation("io.bitrise.trace:trace-sdk:0.0.4")
+    implementation("io.bitrise.trace:trace-sdk:0.0.5")
```

Comment:
nit: do we want to make this 0.0.6 as we're probably going to release that with your cpu changes in the next hour or so?

Reply:
We could, but it would make the test workflows fail. So to be clear, probably I will create a new PR later.